### PR TITLE
Update branches at git_intro_.qmd

### PR DESCRIPTION
**Pull request by RedondoMA**
Added lines on the branch section

### CODE REVIEW

**Review by Estel-Kitsune — APPROVED** [review]
Good summary!

File: git/git_intro.qmd [hunk 1]
```diff
@@ -7,6 +7,13 @@
 ## Git Repositories
 
 ## Git branches
+Once you have cloned a specific git repository locally on your computer, you can navigate and/or create new branches on it using git CLI. Here some examples:
+- To create a branch and switch to it type `git checkout -b branch_name` 
+- To push the newly created branch to the remote repository type `git push -u origin branch_name` 
+- To display all branches on the local and remote repository type `git branch -a` 
+- To switch to some of the displayed branches type `git ckeckout name_of_branch`. **Once a change is commited from that branch, pushing the committed change will be pushed into that specific branch.**
```
Comment (mahesh-panchal) [inline git/git_intro.qmd:14]:
```suggestion
- To switch to one of the displayed branches type `git ckeckout name_of_branch`. **Once a change is committed to that branch, pushing the committed change will be pushed to that specific branch on the remote repository.**
```
Aside from the spelling corrections, does the suggested change say what you were trying to convey? The note was unclear to me.

Comment (RedondoMA) [reply git/git_intro.qmd:14]: Yes, it does. I just wanted to make clear that when you do any change on a specific branch, you push those changes to that specific branch on the remote repo. Thanks for the spelling corrections!

Comment (mahesh-panchal) [reply git/git_intro.qmd:14]: ( You need to commit the changes )

**Comment by RedondoMA** [comment]
I committed your suggestions, Mahesh!

File: git/git_intro.qmd [hunk 2]
```diff
@@ -7,6 +7,13 @@
 ## Git Repositories
 
 ## Git branches
+Once you have cloned a specific git repository locally on your computer, you can navigate and/or create new branches on it using git CLI. Here some examples:
+- To create a branch and switch to it type `git checkout -b branch_name` 
+- To push the newly created branch to the remote repository type `git push -u origin branch_name` 
+- To display all branches on the local and remote repository type `git branch -a` 
+- To switch to one of the displayed branches type `git ckeckout name_of_branch`. **Once a change is committed to that branch, pushing the committed change will be pushed to that specific branch on the remote repository.**
```
Comment (mahesh-panchal) [inline git/git_intro.qmd:14]:
```suggestion
- To switch to one of the displayed branches type `git checkout name_of_branch`. **Once a change is committed to that branch, pushing the committed change will be pushed to that specific branch on the remote repository.**
```

Comment (RedondoMA) [reply git/git_intro.qmd:14]: Thanks!

**Review by mahesh-panchal — APPROVED** [review]

**Pull request merged**